src/minicraft/item/StackableItem.java

```diff
@@ -39,7 +39,7 @@
 	}
 	
 	public int count;
-	// public int maxCount; // TODO I want to implement this later.
+	public int maxCount = 999; // TODO I want to implement this later.

```

Comment:
is this implemented, or did you give it a value but never use it? Also, why is it instance bound if it doesn't change?

Reply:
I was going to implement it; I guess I just forgot.